pyproject.toml

```
@@ -1,25 +1,26 @@
 [build-system]
 requires = [
-    "setuptools>=64",
-    "wheel",
+    "scikit-build-core>=0.9.9",
     # On Windows, use the CasADi vcpkg registry and CMake bundled from MSVC
     "casadi>=3.6.5; platform_system!='Windows'",
     # Note: the version of CasADi as a build-time dependency should be matched
     # cross platforms, so updates to its minimum version here should be accompanied
     # by a version bump in https://github.com/pybamm-team/casadi-vcpkg-registry.
     "cmake; platform_system!='Windows'",
+    # TODO: check if just pybind11 works or not
+    "pybind11",
```

```suggestion
    "pybind11",
```

Now that we have confirmed it, we can remove this line :)